### ISSUE TITLE
'not' keyword isn't being highlighted in Atom

### ISSUE THREAD

**Issue by austincrft** [MagicStack/MagicPython#19]
Default Python syntax:
![image](https://cloud.githubusercontent.com/assets/3616544/10709173/f8b57ba6-79e7-11e5-9d75-fcaa8844c291.png)

MagicPython syntax:
![image](https://cloud.githubusercontent.com/assets/3616544/10709171/d91a03ac-79e7-11e5-9c83-e0cccb1a3e3b.png)

**Comment by 1st1** [comment]
Can't reproduce.  Could you please provide about 10 lines of code before/after those lines?

What color theme are you using?

**Comment by austincrft** [comment]
I imagine that it has to do with the syntax theme because it is happening elsewhere in my code. Here's the info:

Syntax theme:
[solarized-dark-syntax](https://atom.io/themes/solarized-dark-syntax)

Lines above
``` python
def mailing_list_preferences_test():
    email = 'abc@123.com'
    user = User.query.filter_by(email=email).first_or_404()
    form = MailingListForm()

    # Don't fill fields on form submit
    if not form.validate_on_submit():
        form.fill_fields_with_user(user)
```

**Comment by austincrft** [comment]
Just tried it on the other syntax themes that ship with Atom. None of them highlighted the case I gave you just now, but [solarized-light-syntax](https://atom.io/themes/solarized-light-syntax) and [atom-light-syntax](https://atom.io/themes/atom-light-syntax) highlighted some of the `not` keywords properly

**Comment by 1st1** [comment]
Should be fixed in v0.4.34. Please update.

(See for details 9f7e839aca1e067691df2aef42572be636642f69 and d78bdaa88d00cd3a0c387722edc5062a604c6dda)

Thanks a lot for discovering this!

**Comment by austincrft** [comment]
Awesome, thanks!
